common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -122,7 +120,7 @@ public Random current() {
             logger.debug("-Dio.netty.noPreferDirect: {}", !DIRECT_BUFFER_PREFERRED);
         }
 
-        if (!hasUnsafe() && !isAndroid() && !IS_EXPLICIT_NO_UNSAFE) {
+        if (!hasUnsafe() && !isAndroid() && !PlatformDependent0.isExplicitNoUnsafe()) {
```

Comment:
`isExplicitNoUnsafe` should be reflected in `hasUnsafe` ... can we just remove the `!PlatformDependent0.isExplicitNoUnsafe()` part?